.eslintrc.yaml

```diff
@@ -135,16 +135,17 @@ rules:
   }]
   no-tabs: error
   no-trailing-spaces: error
+  object-curly-spacing: [error, always]
   one-var-declaration-per-line: error
   operator-linebreak: [error, after]
   quotes: [error, single, avoid-escape]
   semi: error
   semi-spacing: error
   space-before-blocks: [error, always]
   space-before-function-paren: [error, {
-    "anonymous": "never",
-    "named": "never",
-    "asyncArrow": "always"
+    anonymous: never,
```

Comment:
This was done in response to: https://github.com/nodejs/node/pull/14162#discussion_r126576619

I understand it's not 100% the same property but opening another PR seems a little bureaucratic. It's a purely aesthetic change in line with the comment.